explorer/e2e/anvil/anvil-tabs.ts

```diff
@@ -30,6 +30,28 @@ export const ORGANISM_TYPE_INDEX = 8;
 export const PHENOTYPIC_SEX_INDEX = 9;
 export const REPORTED_ETHNICITY_INDEX = 10;
 
+const anvilDatasetsPreselectedColumns = [
+  { name: "Dataset", sortable: true },
+  { name: "Access", sortable: false },
+  { name: "Identifier", sortable: true },
+  { name: "Consent Group", sortable: true },
+  { name: "Organism Type", sortable: true },
+  { name: "Diagnosis", sortable: true },
+  { name: "Data Modality", sortable: true },
+];
+const anvilDatasetsSelectableColumns = [
+  {
+    name: "Phenotypic Sex",
```

Comment:
Will do, although since this doesn't affect test functionality it might make more sense to do it under #4078 since it would affect all tests! Would that work?

Reply:
Would you be able to go ahead and update these in this PR?

Reply:
Will do!